internal/credentials/kubernetes/ecr/common.go

```diff
@@ -2,4 +2,4 @@ package ecr
 
 import "regexp"
 
-var ecrURLRegex = regexp.MustCompile(`^[0-9]{12}\.dkr\.ecr\.(.+)\.amazonaws\.com/`)
+var ecrURLRegex = regexp.MustCompile(`^(oci://)?[0-9]{12}\.dkr\.ecr\.(.+)\.amazonaws\.com/`)
```

Comment:
```suggestion
var ecrURLRegex = regexp.MustCompile(`^(?:oci://)?[0-9]{12}\.dkr\.ecr\.(.+)\.amazonaws\.com/`)
```

This will optionally match `oci://` without capturing it.